common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -51,7 +53,8 @@ public Stream<MapCategory> getCategories() {
     }
 
     private static final class WynntilsCategory implements MapCategory {
-        private static final MapVisibility DEFAULT_VISIBILITY = new MapVisibility(0, 100, 6);
+        private static final MapVisibility DEFAULT_ICON_VISIBILITY = MapVisibility.DEFAULT_ICON_VISIBILITY;
+        private static final MapVisibility DEFAULT_LABEL_VISIBILITY = MapVisibility.DEFAULT_LABEL_VISIBILITY;
```

Comment:
I was not sure what you meant with https://github.com/magicus/Artemis/pull/24/files/93fb3b1946d821fcefe4dab07576db6766bafc05#r1602959177. How do you think this looks?